omnibus-mainnet.toml

```
@@ -127,3 +132,26 @@ target = ["legacyMarket.Proxy"]
 fromCall.func = "owner"
 func = "setPauseMigration"
 args = [true]
+
+# Governance Councils Configuration
+[var.governance]
+governance_package = "synthetix-governance-satellite:3.0.5"
+governance_snapshot_record_package = "synthetix-snapshot-rewards:3.5.0"
+governance_salt = "synthetix-governance"
+# Wormhole
+governance_wormhole_core = "0x98f3c9e6E3fAce36bAAd05FE09d375Ef1464288B"    # https://docs.wormhole.com/wormhole/reference/constants#core-contracts
+governance_wormhole_relayer = "0x27428DD2d3DD32A4D7f7C497eAaa23130d894911" # https://docs.wormhole.com/wormhole/reference/constants#standard-relayer
+# Snapshot
+snapshot_record = "0x89FCb32F29e509cc42d0C8b6f058C993013A843F" # synthetix debt share https://usecannon.com/packages/synthetix/2.101.2/1-main/interact/synthetix/SynthetixDebtShare/0x89FCb32F29e509cc42d0C8b6f058C993013A843F
+ambassador_weight = "0"                                        # Quadratic Weight
+spartan_weight = "0"                                           # Quadratic Weight
+treasury_weight = "1"                                          # Linear Weight
```

let's keep suffixes for consistency (same for other deployments please)
```suggestion
governance_snapshot_record = "0x89FCb32F29e509cc42d0C8b6f058C993013A843F" # synthetix debt share https://usecannon.com/packages/synthetix/2.101.2/1-main/interact/synthetix/SynthetixDebtShare/0x89FCb32F29e509cc42d0C8b6f058C993013A843F
governance_ambassador_weight = "0"                                        # Quadratic Weight
governance_spartan_weight = "0"                                           # Quadratic Weight
governance_treasury_weight = "1"                                          # Linear Weight
```

---

fixed in #370